views/transportation/tabs.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+// import Icon from 'react-native-vector-icons/Entypo'
+
+import OtherModesView from './otherModes'
+import BusView from './bus'
+// import PauseMenuView from './pause'
+
+export default [
+  {
+    id: 'bus',
+    title: 'Bus',
+    rnVectorIcon: {
+      iconName: 'bus',
+      selectedIconName: 'bus',
```

Comment:
I assumed that we had all of these: https://github.com/oblador/react-native-vector-icons
Noted.